src/renderer/ScreenController.tsx

```diff
@@ -107,6 +107,10 @@ export default class ScreenController extends Component {
   onError(_event: any, [data1, data2]: [string | number, string]) {
     if (this.state.screen === Screens.Login) return
     if (data1 === 0) data1 = ''
+    console.log(data1, data2)
+    // Can get removed as soon as we use a rust core including this fix: https://github.com/deltachat/deltachat-core-rust/pull/1911
```

Comment:
maybe we should create an issue for it, only a comment in the source code gets forgotten quickly.

Reply:
It's not jigga important to remove it, but yeah if you feel like an issue, feel free to open one :)